playbooks/templates/stenographer-config.j2

```diff
@@ -3,6 +3,8 @@
     {
       "PacketsDirectory": "/data/stenographer/{{ item.1 }}/thread0/packets"
     , "IndexDirectory": "/data/stenographer/{{ item.1 }}/thread0/index"
+    , "MaxDirectoryFiles": 30000
```

Comment:
I like this change to make it explicit, but I'd like to change this to Ansible variables. Can you make that change and define them in the "retention block" in [rocknsm_config.dist.yml](https://github.com/rocknsm/rock/blob/devel/playbooks/vars/rocknsm_config.dist.yml#L20)?

Also, note that I changed the base to `devel`. We reserve `master` only for tagged releases where the `devel` branch is bleeding edge.